smart-contract-verifier/smart-contract-verifier-http/src/verification_response.rs

```diff
@@ -23,6 +45,11 @@ pub struct VerificationResult {
     pub abi: Option<String>,
     pub sources: BTreeMap<String, String>,
     pub compiler_settings: String,
+
+    #[serde(default, skip_serializing_if = "Vec::is_empty")]
```

Comment:
why? I don't like the idea that service might not return field even if field is empty.